mlir/utils/performance/perfRunner.py

```diff
@@ -326,7 +326,7 @@ def generateMlirDriverCommandLine(self, rocmlir_gen_flags):
                            '--padding_h', str(self.paddingH),
                            '--padding_w', str(self.paddingW),
                            '--kernel-repeats', str(self.MLIR_N_REPEATS),
-                           f"--perf_config={self.perfConfig}"])
+                           f"--perf_config={self.perfConfig} "])
```

Comment:
Could you remove space changes? (Unless it's important that the space stays there)

Reply:
Yeah, it's important to have a space here.

Reply:
I think it'd be more correct to introduce a leading space to whatever you're appending, please remove.